src/pyodide/types/emscripten.d.ts

```diff
@@ -10,11 +10,24 @@ interface PyodideConfig {
   indexURL?: string;
 }
 
+type InFuncType = () =>
+  | null
+  | undefined
+  | string
+  | ArrayBuffer
+  | Uint8Array
+  | number;
+
 interface API {
   config: PyodideConfig;
   finalizeBootstrap: () => void;
   public_api: Pyodide;
   rawRun: (code: string) => [status: number, err: string];
+  initializeStreams: (
+    stdin?: InFuncType,
```

Comment:
I'd just put `any` since we don't use it.
```suggestion
    stdin?: any,
```